Added docs and antora support (#9)
RedHatInsights/yuptoo: docs/modules/ROOT/pages/ephemeral_environment.adoc

= Ephemeral Environment.

First make sure one has the access to ephemeral cluster and has bonfire and oc binary install in the PATH. For more details regarding getting access and downloading oc and bonfire binary please go through the doc - https://consoledot.pages.redhat.com/docs/dev/getting-started/ephemeral/onboarding.html[Onboarding to the Ephemeral Cluster]

== Run Yuptoo in Ephemeral Environment.

Login to the cluster by grabbing a login command from https://oauth-openshift.apps.c-rh-c-eph.8p0c.p1.openshiftapps.com/oauth/token/request[token request page] for the ephemeral cluster
----
oc login --token=sha256~xxxxxx --server=https://api.c-rh-c-eph.8p0c.p1.openshiftapps.com:6443
----

Run bonfire command to reserve the ephemeral namespace and to deploy yuptoo in that namespace
----
bonfire deploy yuptoo
----

> Above command will return the name of namespace which has been reserved, make a note of it. 

Reviewer: ```suggestion
Note down the name of reserved namespace from the output of above bonfire command. 
```
